tools/Makefile

```diff
@@ -62,10 +62,18 @@ espnet.done: venv
 	. venv/bin/activate && conda install -y pytorch=$(TH_VERSION) -c pytorch
 	. venv/bin/activate && conda install -y matplotlib
 	. venv/bin/activate && pip install -e ..
-	. venv/bin/activate && pip install cupy==$(CUPY_VERSION)
 	touch espnet.done
 endif
 
+ifneq ($(strip $(CUPY_VERSION)),)
+cupy.done: venv
+	. venv/bin/activate && pip install cupy==$(CUPY_VERSION)
+	touch cupy.done
+else
+cupy.done: venv
+	touch cupy.done
```

Comment:
I’m not sure cupy.done is needed if we do not install cupy.

Reply:
resolved in new commit